src/Behat/Behat/Output/Node/EventListener/JUnit/JUnitOutlineStoreListener.php

```diff
@@ -54,12 +55,31 @@ public function __construct(SuitePrinter $suitePrinter)
     public function listenEvent(Formatter $formatter, Event $event, $eventName)
     {
         $this->captureOutlineOnBeforeOutlineEvent($event);
+        if ($event instanceof SuiteTested) {
+            setFormatterFileName($event, $formatter);
```

Comment:
Please write a test for the feature you introduced/changed. You'll see that it is not working and that you are missing an obvious `$this->` here ;-)